src/gsym/types.rs

```diff
@@ -0,0 +1,42 @@
+pub const GSYM_MAGIC: u32 = 0x4753594d;
+pub const GSYM_VERSION: u16 = 1;
+
+/// GSYM File Header
+pub struct Header {
+    pub magic: u32,
+    pub version: u16,
+    pub addr_off_size: u8,
+    pub uuid_size: u8,
+    pub base_address: u64,
+    pub num_addrs: u32,
+    pub strtab_offset: u32,
+    pub strtab_size: u32,
+    pub uuid: [u8; 20],
+}
+
+#[allow(dead_code)]
+pub struct FileInfo {
+    pub directory: u32,
+    pub filename: u32,
+}
```

Comment:
Can we please only add stuff that we actually use? If we need it later then please introduce it later as well.

Reply:
This type will be used in the next patch.  I will move it to the next patch.